specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/cosmos-db.json

```diff
@@ -0,0 +1,7891 @@
+          },
+          "description": "List of Table and their properties."
+        }
+      },
+      "description": "The List operation response, that contains the Table and their properties."
+    },
+    "CassandraKeyspaceListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CassandraKeyspaceGetResults"
+          },
+          "description": "List of Cassandra keyspaces and their properties."
+        }
+      },
+      "description": "The List operation response, that contains the Cassandra keyspaces and their properties."
+    },
+    "CassandraTableListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CassandraTableGetResults"
+          },
+          "description": "List of Cassandra tables and their properties."
+        }
+      },
+      "description": "The List operation response, that contains the Cassandra tables and their properties."
+    },
+    "GremlinDatabaseListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GremlinDatabaseGetResults"
+          },
+          "description": "List of Gremlin databases and their properties."
+        }
+      },
+      "description": "The List operation response, that contains the Gremlin databases and their properties."
+    },
+    "GremlinGraphListResult": {
+      "properties": {
+        "value": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/GremlinGraphGetResults"
+          },
+          "description": "List of graphs and their properties."
+        }
+      },
+      "description": "The List operation response, that contains the graphs and their properties."
+    },
+    "ErrorResponse": {
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      },
+      "description": "Error Response."
+    },
+    "ErrorResponseUpdatedFormat": {
+      "description": "An error response from the service.",
+      "x-ms-external": true,
+      "properties": {
+        "error": {
+          "description": "Details of error response.",
+          "x-ms-external": true,
+          "properties": {
+            "code": {
+              "type": "string",
+              "description": "Error code, intended to be consumed programmatically."
+            },
+            "message": {
+              "type": "string",
+              "description": "Description of the error, intended for display in user interface."
+            },
+            "target": {
```

Comment:
Is this property only applicable for error responses from the new restore paths? Could we just update the existing "ErrorResponse" definition to contain this?

Reply:
The existing version of ErrorResponse does not follow Azure Requirement. 
So to not break the existing apis, I created a new definition.